cedar-policy-cli/src/lib.rs

```diff
@@ -94,10 +94,48 @@ pub enum Commands {
     Link(LinkArgs),
     /// Format a policy set
     Format(FormatArgs),
+    /// Translate JSON schema to custom schema syntax and vice versa (except comments)
+    TranslateSchema(TranslateSchemaArgs),
```

Comment:
Should we also say in the doc comment that the translated schema will be printed on stdout?  or is that implied?

Reply:
I think that's obvious